octobot_trading/modes/scripted_trading_mode/abstract_scripted_trading_mode.py

```diff
@@ -381,30 +381,39 @@ async def _schedule_initialization_call(self):
 
     async def _wait_for_symbol_init(self, symbol, time_frame, timeout):
         # warning: should never be called in backtesting
```

Comment:
Should we add an if not backtesting here?

Reply:
I removed the comment, it can now be called in backtesting

Reply:
👍